test/Serilog.Tests/Capturing/PropertyValueConverterTests.cs

```diff
@@ -113,13 +113,13 @@ public void ByDefaultADestructuredNonNullNullableIsItsValue()
 
         class A
         {
-            public B B { get; set; }
+            public B B { get; set; } = null!;
```

Comment:
Any reason not to just make this (and the similar ones below) into nullable properties? I.e.:

```csharp
public B? B { get; set; }
```

This would match the current semantics of the code.

Using `null!` catches my eye and causes me to dig around more, trying to figure out how the assertion "this property begins life as null but that value can never be observed" could be upheld by the code.

Reply:
good point. done